maquetado/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
+  <nav class="nav-bar">
```

Comment:
use the `header` tag to be more semantic, the `nav` tag is supposed to be inside the header tab to wrap the navigation links. Since there are no nav links here it makes more sense to use a header.